pyphi/models/actual_causation.py

```diff
@@ -131,8 +131,9 @@ class CausalLink(cmp.Orderable):
     up to |PRECISION|, the size of the mechanism is compared.
     """
 
-    def __init__(self, ria):
+    def __init__(self, ria, extended_purview=None):
         self._ria = ria
+        self._extended_purview = extended_purview
```

Comment:
I would cast this to a tuple:
```python
self._extended_purview = tuple(extended_purview)
```

Reply:
if extended_purview is None wouldn't this cast lead to an type error?
Maybe something like ` self._extended_purview = tuple(extended_purview) if extended_purview is not None else None`?